Kentico.Kontent.Management/IManagementClient.cs

```diff
@@ -513,5 +514,13 @@ public interface IManagementClient
         /// <param name="user">Represents an user that is to be modified.</param>
         /// <returns>Returns the modified user.</returns>
         Task<UserModel> ModifyUsersRolesAsync(UserIdentifier identifier, UserModel user);
+        
+        Task<AssetRenditionResponseModel> GetAssetRendition(AssetRenditionIdentifier identifier);
+        
+        Task<IListingResponseModel<AssetRenditionResponseModel>> GetAssetRenditions(Reference assetIdentifier);
```

Comment:
Methods that return `IListingResponseModel` have prefix List instead of Get. Also add `Async` suffix, please.